### PR TITLE
Style Guide - Foundations/Installations: Learning Outcomes and Knowledge Checks

### LINKED ISSUE #23929: Style Guide - Foundations/Installations: Learning Outcomes and Knowledge Checks
# Foundations/Installations Section

This issue is part of the Style Guide Community Project, a way of offering members of The Odin Project community a chance to contribute to our curriculum. Users who wish to take part and contribute to any Style Guide issues should have thoroughly read the [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md) and [The Odin Project contributing guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md).
  
## Description of Issue

This issue ensures the Learning Outcomes and Knowledge Check sections have the correct verbiage and formatting. If one of these sections does not exist in a lesson, it should not be added.

Each applicable lesson for this issue should meet the following requirements:

- The Learning Outcomes section is renamed to "Lesson Overview".
- The Lesson Overview section follows the below format:
	```
	### Lesson Overview

	This section contains a general overview of topics that you will learn in this lesson.

	*   LESSON OVERVIEW ITEMS...
	```
- The Knowledge Check section follows the below format:
	```
	### Knowledge Check

	This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.

	*   KNOWLEDGE CHECKS...
	```
- Each Knowledge Check item follows the below format:
	```
	*   <a class="knowledge-check-link" href="A KNOWLEDGE CHECK URL">A KNOWLEDGE CHECK QUESTION.</a>
	```
- Each Knowledge Check item redirects to a resource mentioned in the lesson, or to a section within the lesson, where the answer can be found. 
	- If it doesn't, note this in your PR, but don't remove the item.
  
## Acceptance Criteria

- [ ] The following lessons are updated according to the issue description above:
	- [ ] [Command Line Basics](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/command-line-basics-web-development-101)

### PR DESCRIPTION
**Pull request by Overlandcoder**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #23929

**2. This PR:**

Change 'Learning Outcomes' to 'Lesson Overview'
Update wording in 'Lesson Overview'
Update wording in 'Knowledge Check'
Add links in 'Knowledge Check'

**3. Additional Information:**

Unable to add an appropriate link to the last question in 'Knowledge Check'

### CODE REVIEW

File: foundations/installations/command_line_basics.md [hunk 1]
```diff
@@ -95,16 +95,16 @@ This section contains helpful links to related content. It isn’t required, so
...
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#uid31">Where will `cd` on its own navigate you to?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#uid30">Where will `cd ..` navigate you to?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-pwd">How do you display the name of the directory you are currently in?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-ls">How do you display the contents of the directory you are currently in?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid7">How do you create a new directory?</a>
+* <a class="knowledge-check-link" href="#create-file">How do you create a new file?</a>
+* <a class="knowledge-check-link" href="https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid9">How do you destroy a directory or file?</a>
 * How do you rename a directory or file?
```
Comment (thatblindgeye) [inline foundations/installations/command_line_basics.md:110]: For this, I would link to the [mv command](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid10) from the assignment link.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Thanks for the updates! I have one requested change below.

File: foundations/installations/command_line_basics.md (hunk 1 above)
Comment (Overlandcoder) [reply foundations/installations/command_line_basics.md]: > For this, I would link to the [mv command](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid10) from the assignment link.

Thanks for helping me out! I've added that link now.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
One last change request. For the `<span>` elements being used here, let's wrap them around some surrounding text. Doesn't have to be a ton of text, just around the general topic they pertain to.

So for example, the `<span id="command-line">` one could wrap around the mention of the command line interface, e.g. `<span id="command-line">command line interface (CLI)</span>`

**Comment by Overlandcoder** [comment]
> One last change request. For the `<span>` elements being used here, let's wrap them around some surrounding text. Doesn't have to be a ton of text, just around the general topic they pertain to.
> 
> So for example, the `<span id="command-line">` one could wrap around the mention of the command line interface, e.g. `<span id="command-line">command line interface (CLI)</span>`

Done! Thanks for pointing that out.

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**